release/README.md

```diff
@@ -153,20 +148,26 @@ take several weeks.
 # Creating a patch release of google-cloud-cpp on an existing release branch
 
 In your development fork:
-* Switch to the existing release branch, e.g. `git checkout v1.17.x`.
-* Bump the version numbers for the patch release
-  * Create a new branch off the release branch
-  * Update the minor version in the top-level `CMakeLists.txt` file.
-  * Run `ci/cloudbuild/build.sh -t check-api-pr` to update both the API
+
+- Switch to the existing release branch, e.g. `git checkout v1.17.x`.
+- Bump the version numbers for the patch release
+  - Create a new branch off the release branch
+  - Update the minor version in the top-level `CMakeLists.txt` file.
+  - Run `ci/cloudbuild/build.sh -t check-api-pr` to update both the API
     baselines and `google/cloud/internal/version_info.h`.
-  * **Send this PR for review and merge it before continuing**
-* Create a new branch off the release branch, which now contains the new patch
+  - **Send this PR for review and merge it before continuing**
+- Create a new branch off the release branch, which now contains the new patch
   version and baseline ABI dumps.
-* Create or cherry-pick commits with the desired changes.
-* Update `CHANGELOG.md` to reflect the changes made.
-* After merging the PR(s) with all the above changes, use the Release UI on
+- Create or cherry-pick commits with the desired changes.
+- Update `CHANGELOG.md` to reflect the changes made.
+- After merging the PR(s) with all the above changes, use the Release UI on
   GitHub to create a pre-release along with a new tag for the release.
-* After review, publish the release.
-* Nudge `coryan@` to update our [vcpkg port].
+- After review, publish the release.
+- Nudge `coryan@` to update our [vcpkg port].
 
+[git-docs]: https://git-scm.com/doc
+[github-branch-settings]: https://github.com/googleapis/google-cloud-cpp/settings/branches
+[github-guides]: https://guides.github.com/
 [vcpkg port]: https://github.com/Microsoft/vcpkg/tree/master/ports/google-cloud-cpp
+[`changelog.md`]: /CHANGELOG.md
```

Comment:
This did more than move the link, it also changed the case.  I am not sure how this renders?

Reply:
It renders fine, I checked in your branch.

Reply:
Yes, the link tag is case independent, so the tool canonicalizes them.